openapi/listings/listings.yaml

```diff
@@ -271,6 +272,49 @@ paths:
             - listing
         - OAuth2Prod:
             - listing
+  '/citations/{id}':
```

Comment:
Do you have an RFC, [API resource vision doc](https://vendasta.jira.com/wiki/spaces/API/pages/999489554/Resource+Visions) or design session doc that outlines how you anticipate partners using this API? It appears that the [Requirements Deep Dive](https://vendasta.jira.com/wiki/spaces/API/pages/1723368157/API+Development+Process#Requirements-Deep-Dive-Product-Team-API-Designer) step may have been missed. 

Based on the description of this PR and LIS-1006 I would have expected us to build an API that lists the citations that match some user supplied filter. Users can tally them up in whatever reporting tool they are using.

```http
GET /citations?filter[listingProfile.id]=AG-123&filter[startAt]=2022-10-01&filter[endAt]=2023-10-02
{
  "data": [
    {
      "id": "c-1",
      "type": "citations",
      "attributes": {
        "breadcrumb": "Home > Page Title",
        "createdAt": "2021-09-20T12:46:46Z",
        "description": "page description",
        "domain": "vendasta.com",
        "searchTerm": "search keyword",
        "title": "Page Title Example",
        "url": "https://vendasta.com/page-title-example.com",
        "matchedFields": [
          "CompanyName",
          "WorkNumber"
        ]
      }
    },
    {
      "id": "c-2",
      "type": "citations",
      "attributes": {
        "breadcrumb": "Home > Page Title",
        "createdAt": "2021-09-20T12:46:46Z",
        "description": "page description",
        "domain": "vendasta.com",
        "searchTerm": "search keyword",
        "title": "Page Title Example",
        "url": "https://vendasta.com/page-title-example.com",
        "matchedFields": [
          "CompanyName",
          "WorkNumber"
        ]
      }
    },
    // For the legacy citations we can add records with just the data we have
    {
      "id": "c-3",
      "type": "citations",
      "attributes": {
        "createdAt": "2021-09-20T00:00:00Z",
        "citationOnDate": 2
      }
    }
  ]
}
```

If displaying the individual data points is not expected to be the primary use case we could offer a citations summary or citation score resource.